bp_fe/src/include/bp_fe_itlb.vh

```diff
@@ -0,0 +1,66 @@
+/*                                                                         
+ * itlb.vh
+ * This file declares the internal structs of the BlackParrot Front
+ * End I-TLB module. For simplicity and flexibility, these structs are
+ * parameterized. Each struct declares its width separately to
+ * prevent pre-processor ordering issues. 
+*/
+
+`ifndef ITLB_VH
+`define ITLB_VH
+
+`ifndef BSG_DEFINES_V
+`define BSG_DEFINES_V
+`include "bsg_defines.v"
+`endif
+
+`ifndef BP_COMMON_FE_BE_IF
+`define BP_COMMON_FE_BE_IF
+`include "bp_common_fe_be_if.vh"
+`endif
+
+import itlb_pkg::*;
+/*
+ * bp_fe_itlb_icache_data_resp_s defines the interface between I-TLB and
+ * I-Cache. The I-TLB sends the Physical Page Number (PPN) to the
+ * I-Cache. The width of PPN is specified by ppn_width_p parameter.
+*/
+  
+`define declare_bp_fe_itlb_icache_data_resp_s(ppn_width_p)\
+  typedef struct packed {                    \
+    logic [ppn_width_p-1:0] ppn;       \
+  } bp_fe_itlb_icache_data_resp_s
+
+`define bp_fe_itlb_icache_width(ppn_width_p) (ppn_width_p)
+/*
+ * In the case of an I-TLB miss, bp_fe_itlb_miss_exception_data_s
+ * struct is passed to the BE for I-TLB miss handling. This struct
+ * includes the requested Virtual Page Nmuber (VPN) whose PTE does not
+ * exist in the I-TLB.
+*/
+
+`define declare_bp_fe_itlb_cmd_s                                       \
+    typedef struct packed {                                            \
```

Comment:
Throughout: 
typedef struct packed
{  
  field1
  field2
}  struct_s;

Reply:
Having { on new line breaks syntax sometimes, need to investigate why this is happening for some structs and not others. other than that this should be ok.

Reply:
That's super weird.  Let me know if you figure out what's going on

Reply:
solved!